gem/templates/yegna/core/article_page.html

```diff
@@ -53,7 +53,9 @@ <h1 class="heading__subheading heading__subheading--watch">{{ page.title}}</h1>
       {% if page.homepage_media %}
         {% for block in page.homepage_media %}
           {% with block.value as media %}
-            <p class="media-player__duration">({{media.duration|time:"i:s"}})</p>
+            {% if media.duration %}
+              <p class="media-player__duration">({{media.duration}})</p>
```

Comment:
I see you removed the duration format here. Was that intentional? i.e `time:"i:s"`

Reply:
@miltontony Yes it is. We need to find a working filter, this currently does not work. Codie is helping with this when she is available.

Reply:
👌